text/0040-pagination-convention.md

```diff
@@ -0,0 +1,47 @@
+- Title: Pagination Convention
+- Start Date: 2021-05-10
+- Specification PR: [#40](https://github.com/meilisearch/specifications/pull/40)
+- MeiliSearch Tracking-Issues:
+
+# Pagination Convention
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Pagination allows the content to be viewed in the form of a sequence in order to facilitate the browsing of its content by not overloading the number of elements to be viewed for the user. Pagination also has the advantage of easily providing the total number of items that remain to be traversed by not overloading the network with all the content that can be loaded.
+
+### II. Motivation
+
+This specification will be used as a convention to implement simple and standard paginations to be set up on API routes such as the list of updates for example. Since the pagination of search results is intrinsically linked to technical implementations of the search engine, this specific case is detailed in its own specification.
+
+### III. Additional Materials
+N/A
+
+### IV. Explanation
+
+##### Requirements
+
+- ✅ Each paginable route must return the `total` number of items that match the query and the number of items returned in the response as `size`.
```

Comment:
What are "each paginable route"? Get and post /search routes?

Reply:
All paginable routes except GET and POST /search. E.g. `/updates`, `/documents`. This specification concerns all routes that can be paged and that are not related to the search endpoints.